---
layout: page
title: Research
description: Joslynn's research
---



#### <u>Genomic Science Leadership Initiative</u>
*This is not published. Co-organized a 3-day workshop introducing participants to the connection and importance of the natural world to molecular biology and genomics. Participants gain hands-on laboratory and computer experience to explore the presence of bacteria (through 16S rRNA) in Colorado and New Mexico environmental samples affected by the Gold King Mine Spill. This experience introduces the latest DNA technologies and current molecular and computational biology methods. Participants learn about science career paths and leadership philosophy/practices.*

[click here for an overview of the 2018 workshop](https://github.com/joslynnlee/joslynnlee.github.io/blob/master/pages/workbook_2018_shortened.pdf)
[click here for an overview of the 2017 workshop](http://maize.jcvi.org/cellgenomics/2017_pcr.php)

#### <u>Using protein active site predictors to classify protein superfamilies</u>
*As a result of high-throughput protein structure initiatives, over 14,400 protein structures
have been solved by structural genomics (SG) centers and participating research groups. While the
totality of SG data represents a tremendous contribution to genomics and structural biology, reliable 
functional information for these proteins is generally lacking. Better functional predictions for
SG proteins will add substantial value to the structural information already obtained. Our method
described herein, Graph Representation of Active Sites for Prediction of Function (GRASP-Func),
predicts quickly and accurately the biochemical function of proteins by representing residues at
the predicted local active site as graphs rather than in Cartesian coordinates. We compare the
GRASP-Func method to our previously reported method, structurally aligned local sites of activity
(SALSA), using the ribulose phosphate binding barrel (RPBB), 6-hairpin glycosidase (6-HG), and
Concanavalin A-like Lectins/Glucanase (CAL/G) superfamilies as test cases. In each of the superfa-
milies, SALSA and the much faster method GRASP-Func yield similar correct classification of pre-
viously characterized proteins, providing a validated benchmark for the new method. In addition,
we analyzed SG proteins using our SALSA and GRASP-Func methods to predict function. Forty-
one SG proteins in the RPBB superfamily, nine SG proteins in the 6-HG superfamily, and one SG
protein in the CAL/G superfamily were successfully classified into one of the functional families in
their respective superfamily by both methods. This improved, faster, validated computational
method can yield more reliable predictions of function that can be used for a wide variety of appli-
cations by the community.*

[click here for the paper "Functional classification of protein
structures by local structure matching in graph representation"]({{ BASE_PATH}}/pages/working_papers/GRASPFunc.pdf)

<!-- Note: this is how to write a comment in HTML. Everything in here won't show up on your webpage.-->

<!--
To increase the size of the title, use fewer # in front of the paper title.
To decrease the size of the title, use more #. 
To remove the italics, remove the * before and after the description
To remove the underline from the title, remove the <u> tags (<u> and </u>)
-->
